tests/0.2.2/post.yaml

```diff
@@ -0,0 +1,132 @@
+---
+version: 0.2.2
+function: post
+tests:
+  - cases:
+    - name: reports log aggregation
+      client: client
+      function: post
+      aggregator: log_path
+      data:
+        "my.path":
+          - 3
+          - 1
+        "another.path":
+          - 0
+          - 0
+          - 1
+          - 2
+      expected_data:
+        - logger_name: "my.path"
+          counts:
+            debugs: 3
+            infos: 1
+        - logger_name: "another.path"
+          counts:
+            warns: 1
+            errors: 2
+      client_overrides:
+        collect_sync_interval: 600
+
+    - name: reports context shape aggregation
+      client: client
+      function: post
+      aggregator: context_shape
+      data:
+        user:
+          name: "Michael"
+          age: 38
+          human: true
+        role:
+          name: "developer"
+          admin: false
+          salary: 15.75
+          permissions:
+            - "read"
+            - "write"
+      expected_data:
+        - name: "user"
+          field_types:
+            name: 2
+            age: 1
+            human: 5
+        - name: "role"
+          field_types:
+            name: 2
+            admin: 5
+            salary: 4
+            permissions: 10
+      client_overrides:
+        collect_sync_interval: 1000
+        context_upload_mode: :shape_only
+
+    - name: reports evaluation summary
+      client: client
+      function: post
+      aggregator: evaluation_summary
+      data:
+        - "my-test-key"
+        - "feature-flag.integer"
+        - "my-string-list-key"
+        - "feature-flag.integer"
+      expected_data:
+        - key: "my-test-key"
+          type: CONFIG
+          value: "my-test-value"
+          value_type: string
+          count: 1
+          summary:
+            config_row_index: 0
+            conditional_value_index: 1
+        - key: "my-string-list-key"
+          type: CONFIG
+          value:
+            - "a"
+            - "b"
+            - "c"
+          value_type: string_list
+          summary:
+            config_row_index: 0
+            conditional_value_index: 0
+          count: 1
+        - key: "feature-flag.integer"
+          type: FEATURE_FLAG
+          value: 3
+          value_type: int
+          count: 2
+          summary:
+            config_row_index: 0
+            conditional_value_index: 1
+      client_overrides:
+        collect_sync_interval: 1000
+
+    - name: reports example contexts
+      client: client
+      function: post
+      aggregator: example_contexts
+      data:
+        - user:
+            name: "michael"
+            age: 38,
```

Comment:
This `,` typo should be removed. Ruby's YAML implementation treats it as `38` but Node's YAML implementation treats it as the string `"38,"`